### PR TITLE
Elcomsoft.CloudeXplorer version 2.32.37098.6840

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102153)

### CODE REVIEW

File: manifests/e/Elcomsoft/ElcomsoftCloudeXplorer/2.32.37098.6840/Elcomsoft.ElcomsoftCloudeXplorer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftCloudeXplorer
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftCloudeXplorer/2.32.37098.6840/Elcomsoft.ElcomsoftCloudeXplorer.installer.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.CloudeXplorer
```

File: manifests/e/Elcomsoft/ElcomsoftCloudeXplorer/2.32.37098.6840/Elcomsoft.ElcomsoftCloudeXplorer.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftCloudeXplorer
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftCloudeXplorer/2.32.37098.6840/Elcomsoft.ElcomsoftCloudeXplorer.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.CloudeXplorer
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with:
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\e\Elcomsoft\ElcomsoftCloudeXplorer\2.32.37098.6840', expected 
'manifests\e\Elcomsoft\CloudeXplorer\2.32.37098.6840' 
```

There's an extra `Elcomsoft` in the path, before `CloudeXplorer`.

**Comment by upintheairsheep** [comment]
Seriously GitHub, just add the multiple item moving feature.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**